### PR TITLE
Betting 413. ODDS class

### PR DESCRIPTION
**Pull request by Andrei-Masilevich**
1. Extended fraction class
2. Added new odds class
3. unit tests

### CODE REVIEW

File: tests/utests/fraction_tests.cpp [hunk 1]
```diff
@@ -86,5 +88,21 @@ BOOST_AUTO_TEST_CASE(asset_with_fractional_operations_check)
     }
 }
 
+BOOST_AUTO_TEST_CASE(fraction_simplify_check)
+{
+    BOOST_CHECK(utils::make_fraction(20'000, 100'000).simplify() == utils::make_fraction(1, 5));
+
+    BOOST_CHECK(utils::make_fraction(8, 12).simplify() == utils::make_fraction(2, 3));
+
+    BOOST_CHECK(utils::make_fraction(2, 3).simplify() == utils::make_fraction(2, 3));
```
Comment (alexplevako) [inline tests/utests/fraction_tests.cpp]: add tests for negative `nominator` and `denominator`

File: tests/utests/odds_tests.cpp [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+#include "defines.hpp"
+
+#include <limits>
+
+namespace odds_tests {
+using namespace scorum;
+using namespace scorum::protocol;
+
+BOOST_AUTO_TEST_SUITE(odds_tests)
```
Comment (alexplevako) [inline tests/utests/odds_tests.cpp]: add tests for negative nominator and denominator

**Comment by alexplevako** [comment]
for merging use *squash* with commit message something like: bloc413 -- create odds class
